src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/alerts-classic-vs-workflows.mdx

```diff
@@ -0,0 +1,912 @@
+---
+title: Notification migration 
+metaDescription: Migrating notifications from alerts and applied intelligence 
+redirects: 
+---
+
+At the end of January 2023, the classic notification system for alerts and applied intelligence will be deprecated and we will be implementing the use of workflows instead. This is good news because workflows is an improved, flexible notification system that helps your team learn about potential errors faster so you can take immediate action.
+
+What does [this migration](https://discuss.newrelic.com/t/plan-to-upgrade-alert-notification-channels-to-workflows-and-destinations/188205) mean for your team? In our previous notification model, your team would create an alert condition with different thresholds and parameters. If that condition was associated with a specific policy, was violated, and you wanted your team to know about it right away, you would add **notification settings**. Our notification settings would tell New Relic what data to send where.  
+
+Now, we've added workflows. Workflows are flexible and they don't bind your notification options or your ability to organize alert conditions into polices. For example, you can filter your workflows by a team tag, or check for a "priority" to route issues to the most appropriate destination.
+
+## Configuring custom payloads [#configure-custom-payloads]
+
+To move from notifications to workflows your team might need to make some tweaks to your custom payloads. Workflows still function in the same way as notifications in that, when a condition is violated then a notification is sent to a webhook and when it is sent, it goes with its custom payload. The migration from notification channels to workflows requires changing some of the terminology in this payload.
+
+The following table provides a translation between what the webhook payload names used in our classic notification system and their new, corresponding names in the issue payload.  
+
+For many keys, the issue payload may contain a list of values.
+
+<table>
+
+  <thead>
+    <tr>
+      <th>
+        **Alerts (classic) Name**
+      </th>
+
+      <th>
+        **Alerts (classic) Variable**
+      </th>
+
+      <th>
+        **Alerts (classic) Data Type**
+      </th>
+
+      <th>
+        **Issues Data Type**
+      </th>
```

Comment:
Let's drop these.  For the webhook, they need to be consistent, which means that in many cases these are wrong or at least confusing/misleading  There isn't a good page documenting the Issues model, so it may be worth adding a blurb/section above describing how the Issues aggregate data across all incidents, so most of their attributes are actually lists.